db/migrate/20220531180853_create_roles.rb

```diff
@@ -4,6 +4,7 @@ class CreateRoles < ActiveRecord::Migration[7.0]
   def change
     create_table :roles do |t|
       t.string :name, null: false
+      t.string :color, default: '', null: false
```

Comment:
default: '' allows empty records in the db which isn't what we want

Reply:
The problem was with rubocop, I haven't thought about it.
Should I disable the convention for that line?

Reply:
Have you faced something like this when setting `null: false`?

Reply:
Hmm that should be fine then